packages/salesforcedx-utils-vscode/src/types/index.ts

```diff
@@ -26,6 +26,7 @@ export interface ContinueResponse<T> {
 
 export interface CancelResponse {
   type: 'CANCEL';
+  msg?: string;
```

Comment:
I briefly discussed adding an optional notification error message on CancelResponses with @vazexqi a while back. I thought it would be useful to have in this scenario. Let me know if this should be removed